utilities/check_mrTrix_Version.m

```diff
@@ -0,0 +1,19 @@
+function mrTrixVersion = check_mrTrix_Version
```

Comment:
This doesn't look like it has been fixed. For this function to run properly we need to first call AFQ_mrtrix_set_ld_path. However AFQ_mrtrix_set_ld_path takes as an input, the output of the present function. Gari, did this work for you on linux?